hoot-services/src/main/java/hoot/services/controllers/grail/WaitOverpassUpdate.java

```diff
@@ -84,43 +83,38 @@ public CommandResult execute() {
             long timeSpent = 0;
             String url = null;
 
-            long lastId = DbUtils.getLastPushedId(jobId);
-
-            String query = "[out:json];"
+            String lastId = DbUtils.getLastPushedId(jobId);
+            String time = LocalDateTime.now().minusMinutes(5).toString(); // get 5 minutes before in case the update happened before this command was run
+            String query = "[out:csv(::changeset)][bbox:{{bbox}}];"
                     + "("
-                    + "node(id:" + lastId + ");"
-                    + "way(id:" + lastId + ");"
-                    + "rel(id:" + lastId + ");"
+                    + "node(newer:\"" + time + "\");"
```

Comment:
let's add ways and relations to the query since the last changeset might not contain nodes.